modules/importer/src/runner/cwe/walker.rs

```diff
@@ -0,0 +1,102 @@
+use crate::runner::common::Error;
+use crate::runner::report::{Phase, ReportBuilder};
+use std::io::{Cursor, Read};
+use std::sync::Arc;
+use tokio::sync::Mutex;
+use tokio_util::bytes::Buf;
+use tracing::instrument;
+use trustify_entity::labels::Labels;
+use trustify_module_ingestor::service::{Format, IngestorService};
+use zip::ZipArchive;
+
+#[derive(Clone, Debug, Default, serde::Serialize, serde::Deserialize)]
+pub struct LastModified(Option<String>);
+
+pub struct CweWalker {
+    continuation: LastModified,
+    source: String,
+    ingestor: IngestorService,
+    report: Arc<Mutex<ReportBuilder>>,
+}
+
+impl CweWalker {
+    pub fn new(
+        source: impl Into<String>,
+        ingestor: IngestorService,
+        report: Arc<Mutex<ReportBuilder>>,
+    ) -> Self {
+        Self {
+            continuation: LastModified(None),
+            source: source.into(),
+            ingestor,
+            report,
+        }
+    }
+
+    /// Set a continuation token from a previous run.
+    pub fn continuation(mut self, continuation: LastModified) -> Self {
+        self.continuation = continuation;
+        self
+    }
+
+    /// Run the walker
+    #[instrument(skip(self), ret)]
+    pub async fn run(self) -> Result<LastModified, Error> {
+        let response = reqwest::get(&self.source).await?;
+
+        let last_modified = response
+            .headers()
+            .get("Last-Modified")
```

Comment:
IIRC headers are case insensitive, and there are headers definitions from the `http` create. Does it make sense to use that?